app/ml/base.py

```diff
@@ -0,0 +1,55 @@
+from app.schemas.ml import ModelSummaryResponse
+
+
+class BaseMl:
+    BATCH_SIZE = 32
+    TRAIN_EPOCHS = 2
+
+    def __init__(self, data_dir):
+        self.data_dir = data_dir
+        self._model = None  # Model is lazy-loaded.
```

Comment:
I wanted the model to lazy load, and also to have its most recent build/training available in memory for any number of calls to the analyze route (w/o rebuilding). Decidedly the analyze route will store the images, but will not include them with the dataset trained against. I could see expanding this project out to do more with the analyzed images (as well as a better way to persist data model).